app/views/layouts/application.html.erb

```diff
@@ -80,10 +80,10 @@
   <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/leaflet-search@2.4.0/dist/leaflet-search.min.css">
   </head>
 
-  <body>
-
     <%= render :partial => 'layouts/header' %>
 
+   <%= render :partial => 'app/views/layouts/loading_page.html.erb' %>
```

Comment:
Per my last comment, try changing this to `<%= render :partial => 'layouts/loading_page' %> `

The path you use above is throwing an error for me.